.hlint.yaml

```diff
@@ -0,0 +1,10 @@
+- modules:
+  - { name: Ouroboros.Consensus.MiniProtocol.ChainSync.Client
+    , importStyle: explicitOrQualified
+    , as: [CSC, CsClient, CSClient]
+    , message: "Exports fields with very common names"
+    }
+
+- ignore: {}
+
+- warn: {name: Ouroboros.Consensus.MiniProtocol.ChainSync.Client should be imported qualified or with an explicit import list}
```

Comment:
It is a pain in the neck that we cannot use globs, I would have written `* should be imported *` and then we would get all the warnings of this kind always, but it can't be done